suite-native/module-accounts-import/src/screens/AccountsImportScreen.tsx

```diff
@@ -81,10 +82,10 @@ export const AccountsImportScreen = ({
 
     return (
         <Screen header={<AccountImportHeader activeStep={accountInfo ? 4 : 3} />}>
-            {accountInfo ? (
+            {isLoaded && accountInfo ? (
                 <AccountImportSummary accountInfo={accountInfo} networkSymbol={networkSymbol} />
             ) : (
-                <AccountImportLoader />
+                <AccountImportLoader onLoad={() => setIsLoaded(true)} />
```

Comment:
Shouldn't this be separated screen? I kind of miss transition between this loader and summary. Please verify with Katka or UI/UX guys.

Reply:
Screen divided into two in 2f91908c1